lib/active_record_florder.rb

```diff
@@ -55,6 +55,14 @@ def step(value)
     def get_step
       @step || 2**16
     end
+
+    def populate(value)
+      @populate = value
+    end
+
+    def get_populate
```

Comment:
`attr_accessible` is Rails method so it's not available in this class at all. I'd like to use `attr_accessor` but I can't figure out how to make this work.